types/node/package.json

```diff
@@ -1,15 +1,15 @@
 {
     "private": true,
     "name": "@types/node",
-    "version": "20.14.9999",
+    "version": "22.0.9999",
```

Comment:
Just checking, this says `22.0`; should it be a patch like `22.5`? Or does this really only contain things added in 22.0?

Reply:
I checked the PR and I think it does all check out for 22.0.